### ISSUE TITLE
pip and windows

### ISSUE THREAD

**Issue by andenix** [FutureLinkCorporation/fann2#12]
Could you mention on the pip page that its very hard to compile the bindings on windows.

Could you also add a link there:
    http://www.lfd.uci.edu/~gohlke/pythonlibs/#fann2

A web site that has a working binary of python fann2 v1.07

Thanks.

**Comment by tty02-fl** [comment]
I agree this will help the windows users, because we use a multi-platform solution that run even in a Raspbian or MacOS sacrificing the easy setups.

**Comment by harryzheng1991** [comment]
Are you guys referring to thread #11 ? I assume it has something to do with the compiler. I also tried to use python installer: python setup.py install. with the python package I downloaded. It gives the same error message. not sure it is only a pip issue. But sure thing, I can post it on there later this week. Might give that binary a try too. hopefully it will work... finger crossed. 

The other thing I found in other thread is that some people were talking about compile c code into dll and lib file and put them somewhere in the machine. I tried that but it didn't help. Am I supposed to do that? From the log file, it doesn't look like the setup script looked for those dll and lib files....thanks!

**Comment by andenix** [comment]
I am not sure, I did manage to go through the most steps, but when it tried to link the library I get tons or error about the wrong entry points for function. And somehow it looks for doublefan.lib, however its always called fandouble.lib/dll when you compile it.
I used Fann2.20 (compiled under VS2010). Windows 7 64. however I use 32bit Python and 32bit compiler.
For pip I use gcc that comes with latest mingw.
